Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/competition/BlueLeft.java

```diff
@@ -0,0 +1,94 @@
+package org.firstinspires.ftc.team6220_PowerPlay.competition;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.team6220_PowerPlay.BaseAutonomous;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+import org.openftc.easyopencv.OpenCvCamera;
+import org.openftc.easyopencv.OpenCvCameraRotation;
+
+@Autonomous(name = "BlueLeft")
+public class BlueLeft extends BaseAutonomous {
```

Comment:
Shouldn't BlueLeft and RedLeft be the same? Why do you have separate files?

Reply:
We have separate auton method for each quadrant of the field to help drivers

NOTE:

We might generalize the code inside these classes into a method soon